src/main/java/frc/trigon/robot/subsystems/roller/placeholderroller/PLACEHOLDERRollerIO.java

```diff
@@ -0,0 +1,48 @@
+package frc.trigon.robot.subsystems.roller.placeholderroller;
+
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.trigon.robot.subsystems.roller.RollerIO;
+import frc.trigon.robot.subsystems.roller.RollerInputsAutoLogged;
+
+public class PLACEHOLDERRollerIO extends RollerIO {
+    private final TalonFX motor = PLACEHOLDERRollerConstants.MOTOR;
+    private final DigitalInput infraredSensor = PLACEHOLDERRollerConstants.INFRARED_SENSOR;
+    private final VelocityTorqueCurrentFOC currentRequest = new VelocityTorqueCurrentFOC(0);
+
+    @Override
+    protected void updateInputs(RollerInputsAutoLogged inputs) {
+        inputs.motorVoltage = getVoltage();
```

Comment:
Use BaseStatusSignal.refreshAll to refresh all the status signals more efficiently.